todo_app/app.py

```diff
@@ -8,4 +9,69 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    """
+    Returns the list of saved todo items and sorts by status in descending order. Sorts the item list by status ("Not Started", then "Completed")
+
+    """
+    items = get_items()
+    sorted_items = sorted(items, key=lambda item: item["status"], reverse=True)
+    return render_template("index.html", items=sorted_items)
+
+@app.route('/item', methods = ['POST', 'GET'])
+def add_task():
+    """
+    Returns the list of saved todo items. Redirects the user back to the index page
+
+    """
+    if request.method == "POST":
+        new_item = request.form.get("title")
+        add_item(new_item)
+        return redirect(url_for('index'))
+    else:
+        return render_template("index.html")
+
+@app.route('/update', methods = ['POST', 'GET'])
```

Comment:
`/update` reads a little like you added this before `/inprogress` since that could also fall under the same name; would switching this to `/complete` now make more sense?

Reply:
Incorporated suggestion